compiler/tools/fpp-to-cpp/src/main/scala/fpp-to-cpp.scala

```diff
@@ -62,6 +62,7 @@ object FPPToCpp {
           options.guardPrefix,
           options.pathPrefixes,
           options.defaultStringSize,
+          options.template,
```

Comment:
It's OK to use a boolean flag for the command-line option, but let's convert this to a mode variable here. See above.

Reply:
With two visitors I don't think we need a mode variable.